pages/common/pathchk.md

```diff
@@ -5,16 +5,16 @@
 
 - Check pathames for validity in the current system:
 
-`pathchk {{path1 path2 …}}`
+`pathchk {{path1}} {{path2}}`
```

Comment:
It almost looks like the original is standard syntax for multiple options (which I'm a fan of 👍🏻).  If it is, I'd like to open a new PR to standardize this across the other pages (and I'll fix it here as well).  Is it?

Reply:
I like it too. It should render to the exact same result and looks cleaner.

Reply:
Well, not _exactly_ the same, depending on your client.  Mine, the node client, underlines text in token syntax, so the difference between the old one (which I prefer) and the new modification I made (which I'll revert) is that the old one underlines the space because the space is included between the handlebars, while in the newer version, the space is outside of them.

Reply:
Generally in other pages I've done `{{thing1}} {{thing2}} {{thingN}}` personally

I think it's dependent on context